manifest.json

```diff
@@ -237,6 +235,37 @@
             }
           }
         ]
+      },
+      "includeTwitterMetaTags": {
+        "oneOf": [
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [false]
+              }
+            }
+          },
+          {
```

Comment:
I don't think this is working as you were expecting. When I toggle this to `false` in the interface, the form fields below are not hidden.

Reply:
My bad. Forgot to re-link the app after the last update.